src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerProfile.java

```diff
@@ -39,10 +41,12 @@
 
     @Column(name = "first_name")
     @Size(max = 128)
+    @Pattern(regexp = NAME_REGEX)
```

Comment:
I think you need  to add this in `CaseWorkerProfile.java` in `client/domain package`

Reply:
@SujeethKumarAri  - will make the change, can I ask why is there two?

Reply:
One is for Database Entity Object other one is for client domain. I know it should have been given two different names. Initially different people were working on different features, so when doing a merge we had same name but different packages

Reply:
Would it be worth refactoring this class name in this PR?

Reply:
Addressed here: https://github.com/hmcts/rd-caseworker-ref-api/pull/232/commits/c202fa64288a40583cdd76c4211b8477434266d3

Reply:
thanks